hisensetv/__init__.py

```diff
@@ -226,6 +239,60 @@ def send_key(self, keyname: str):
         """
         self._call_service(service="remote_service", action="sendkey", payload=keyname)
 
+    def launch_app(self, app: str):
+
+        if app == "amazon":
+            launch = {"name": "Amazon", "urlType": 37, "storeType": 0, "url": "amazon"}
+
+        if app == "netflix":
+            launch = {
+                "name": "Netflix",
+                "urlType": 37,
+                "storeType": 0,
+                "url": "netflix",
+            }
+
+        elif app == "youtube":
+            launch = {
+                "name": "YouTube",
+                "urlType": 37,
+                "storeType": 0,
+                "url": "youtube",
+            }
+
+        else:
+            pass
+
+        self._call_service(service="ui_service", action="launchapp", payload=launch)
+
+    def change_source(self, id: str):
```

Comment:
Can you add a docstring here, or prefix with `_` to indicate that this is an internal function?